### DISCUSSION TITLE
Need information on argo metrics

### DISCUSSION THREAD

**Discussion by Roma110** [argoproj/argo-workflows#5291] [Q&A]
Hello,
  We are trying to deploy one sample workflow to check metrics but the value of the below metrics we are not able to understand:
argo_workflows_operation_duration_seconds_sum
argo_workflows_operation_duration_seconds_bucket
argo_workflows_operation_duration_seconds_count

We are referring this  document : [Github](https://github.com/argoproj/argo-workflows/blob/master/examples/custom-metrics.yaml)
We didn't get information about some metrics in argo documentation: [Argo](https://argoproj.github.io/argo-workflows/metrics/)

**Comment by alexec** [comment]
@simster7 ? do we have docs?

**Comment by siretart** [comment]
Well, the documentation is quite terse:

> https://argoproj.github.io/argo-workflows/metrics/#argo_workflows_operation_duration_seconds
> A histogram of durations of operations.



@Joibel do you happen to remember what kind of "operation" this metric it talking about?

**Comment by Joibel** [comment]
This is a workflow-controller reconciliation loop time, so a measure of the performance of the controller. I'll do a quick docs PR to see if I can make it make more sense.

**Comment by Joibel** [comment]
#12282 - please let me know if the wording doesn't make sense or can be clearer

**Reply by siretart** [reply]
I agree, that wording makes much more sense to me. What confused me is that for our installation, the metric would frequently cap out at 3s, which is because of https://github.com/argoproj/argo-workflows/pull/11780 -- maybe a candidate for backporting to a future 3.4.x release?
